api-update/build.sbt

```diff
@@ -0,0 +1,69 @@
+import Dependencies._
+import sbt.Keys.fork
+
+ThisBuild / scalaVersion := "2.13.1"
+ThisBuild / version := "0.1.0-SNAPSHOT"
+ThisBuild / organization := "uk.gov.nationalarchives"
+ThisBuild / organizationName := "api-update"
+
+
+lazy val root = project
+  .in(file("."))
+  .settings(commonSettings)
+  .disablePlugins(AssemblyPlugin)
+  .aggregate(
+    common
+  )
+
+lazy val common = project
+  .settings(
+    name := "common",
+    commonSettings,
+    testSettings,
+    libraryDependencies ++= commonDependencies
+  )
+  .disablePlugins(AssemblyPlugin)
+
+lazy val antivirus = project
+  .settings(
+    name := "antivirus",
+    commonSettings,
+    assemblySettings,
+    testSettings,
+    libraryDependencies ++= commonDependencies
+  )
+  .dependsOn(
+    common % "test->test;compile->compile"
+  )
+
+lazy val testSettings = Seq(
```

Comment:
See comment above about possibly using a config file.

Reply:
As above, it's probably possible, but it would more than likely be complicated and slow and isn't worth it to avoid setting test environment variables.